### ISSUE TITLE
search field not working properly when using a nested view layout

### ISSUE THREAD

**Issue by choper725** [abap2UI5/abap2UI5#474]
hi @oblomov-dev 
search field not working correctly when using nested view layout..

see demo program
[search field demo on nested view.txt](https://github.com/abap2UI5/abap2UI5/files/12304651/search.field.demo.on.nested.view.txt)

dont ask me how i got it because i cant explain :) but, when commenting the controller definition in the http_handler for the nested view, the search field works as expected..
![image](https://github.com/abap2UI5/abap2UI5/assets/129428114/81a35a23-d967-497f-86f2-5e894b82f7e8)

any ideas ?? commenting this out as you surly know (you wrote it 🥇 ), makes the nested view unfunctional..

**Comment by oblomov-dev** [comment]
Yes I remember ;) good to see that you go into the details, finding these bugs is important!

I am also not a full UI5 expert and developer so i always discover new things, don’t know where this behaviors does come from. Maybe ui5 don’t like that the same controller is used multiple times? Did not find a clone function for controllers, so I created the controller in the beginning just two times now and import a second nest controller in the nest generation:
<img width="500" alt="image" src="https://github.com/abap2UI5/abap2UI5/assets/102328295/32844c88-9452-4f05-ad3b-1b20f0ce6979">

Updated the main branch, It seems to work now again.
